test/e2e/tests/test_api_keys.py

```
@@ -904,87 +903,36 @@ class TestEphemeralKeyCleanup:
     def deployment_namespace(self) -> str:
         return os.environ.get("DEPLOYMENT_NAMESPACE", "opendatahub")
 
-    def test_cronjob_exists_and_configured(self, deployment_namespace: str):
-        """Verify the maas-api-key-cleanup CronJob exists with expected configuration."""
+    def test_cleanup_interval_configured(self, deployment_namespace: str):
+        """Verify maas-api Deployment has CLEANUP_INTERVAL_MINUTES configured."""
         import subprocess as sp
 
         result = sp.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)